lightbulb/commands/commands.py

```diff
@@ -158,8 +189,10 @@ class CommandMeta(type):
             is subclassed. I.e. subclassing :obj:`SlashCommand` sets this parameter to :obj:`hikari.CommandType.SLASH`.
         name (:obj:`str`, required): The name of the command.
         description (:obj:`str`, optional): The description of the command. Only required for slash commands.
+        localize (:obj:`bool`, optional): Whether to localize the command's name and description. If :obj:`true`,
+            then the ``name`` and ``description`` arguments will instead be interpreted as localization keys from
+            which the actual name and description will be retrieved from. Defaults to :obj:`False`.
```

Comment:
```suggestion
            then the ``name`` and ``description`` arguments will instead be interpreted as localization keys which
            are used to look up the actual name and description. Defaults to :obj:`False`.
```

My suggestion probably isn't much better here but that last "from" in your change here isn't needed